src/antidote_crdt_orset.erl

```diff
@@ -108,33 +108,33 @@ value({fragment, Elem}, ORSet) ->
 value({tokens, Elem}, ORSet) ->
     case orddict:find(Elem, ORSet) of
         error ->
-            orddict:new();
+            [];
```

Comment:
Yes, that would be more precise. `Token::any()` means that any value can be in the list and `Token::` is just a name for documenting what the value represents.

If we are already fixing the types: The types for downstream operations is also wrong. 

```
% Changed types:
-opaque orset() :: orddict:orddict(member(), [token()]).

-type orset_downstream_op() ::
      {add, member_with_tokens()}
    | {remove, member_with_tokens()}
    | {add_all, [member_with_tokens()]}
    | {remove_all, [member_with_tokens()]}.

-type member() :: term().
-type token() :: binary().
-type member_with_tokens() :: {member(), [token()]}.

% Spec for value
-spec value({fragment, member()}, orset()) -> orset();
           ({tokens, member()}, orset()) -> [token()].

% spec fo downstream:
 -spec downstream(orset_op(), orset()) -> {ok, orset_downstream_op()}.

% spec for update:
-spec update(orset_downstream_op(), orset()) -> {ok, orset()}.
```

Or we can fix this later ...